### PR TITLE
Add rubygems.org icon to email template

### PR DESCRIPTION
**Pull request by sonalkr132**
Screenshot from 2019-06-26 12-17-46
Also fixes weird white space next to left img folds.

### CODE REVIEW

File: app/views/layouts/mailer.html.erb [hunk 1]
```diff
@@ -289,7 +290,22 @@
...
 										<table width="100%" border="0" cellspacing="0" cellpadding="0" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%"><tr><td height="30" class="spacer" style="font-size:0pt; line-height:0pt; text-align:center; width:100%; min-width:100%">&nbsp;</td></tr></table>
 
-										<div class="h3-2-center" style="color:#1e1e1e; font-family:Arial, sans-serif; min-width:auto !important; font-size:20px; line-height:26px; text-align:center; letter-spacing:5px">RUBYGEMS&#65279.ORG</div>
+										<table width="100%" border="0" cellspacing="0" cellpadding="0">
+										  <tr>
+										    <td align="center">
+										        <table border="0" cellspacing="0" cellpadding="0">
+										          <tr>
+										            <td class="img-center" style="font-size:0pt; line-height:0pt; text-align:center" width="60">
+										              <img src=<%= image_url("rubygems_logo_black.png") %> border="0" width="40" height="40" alt="" />
```
Comment (spk) [inline app/views/layouts/mailer.html.erb]: This is fixed by most email client but src attr value should be in quotes

``` diff
--- app/views/layouts/mailer.html.erb
+++ app/views/layouts/mailer.html.erb
@@ -296,7 +296,7 @@
                                                                                        <table border="0" cellspacing="0" cellpadding="0">
                                                                                          <tr>
                                                                                            <td class="img-center" style="font-size:0pt; line-height:0pt; text-align:center" width="60">
-                                                                                             <img src=<%= image_url("rubygems_logo_black.png") %> border="0" width="40" height="40" alt="" />
+                                                                                             <img src="<%= image_url("rubygems_logo_black.png") %>" border="0" width="40" height="40" alt="" />
                                                                                            </td>
                                                                                            <td class="img-center" style="font-size:0pt; line-height:0pt; text-align:center" width="60">
                                                                                              <div class="h3-2-center" style="color:#1e1e1e; font-family:Arial, sans-serif; min-width:auto !important; font-size:25px; line-height:46px; text-align:center; letter-spacing:5px">RUBYGEMS&#65279.ORG</div>
```

File: app/views/layouts/mailer.html.erb [hunk 2]
```diff
@@ -322,7 +338,7 @@
 												<td>
 													<table width="100%" border="0" cellspacing="0" cellpadding="0">
 														<tr>
-															<td class="img" style="font-size:0pt; line-height:0pt; text-align:left" bgcolor="#ffffff" width="27"><img src=<%= image_url("top_left.jpg") %> border="0" width="27" height="27" alt="" /></td>
+															<td class="img" style="font-size:0pt; line-height:0pt; text-align:left" bgcolor="#ffffff" width="27"><img src=<%= image_url("top_left.jpg") %> border="0" width="28" height="27" alt="" /></td>
```
Comment (spk) [inline app/views/layouts/mailer.html.erb]: Same for img src value in quote here

File: app/views/layouts/mailer.html.erb [hunk 3]
```diff
@@ -404,7 +420,7 @@
 													</table>
 													<table width="100%" border="0" cellspacing="0" cellpadding="0">
 														<tr>
-															<td class="img" style="font-size:0pt; line-height:0pt; text-align:left" bgcolor="#ffffff" width="27"><img src=<%= image_url("bot_left.jpg") %> border="0" width="27" height="27" alt="" /></td>
+															<td class="img" style="font-size:0pt; line-height:0pt; text-align:left" bgcolor="#ffffff" width="27"><img src=<%= image_url("bot_left.jpg") %> border="0" width="28" height="27" alt="" /></td>
```
Comment (spk) [inline app/views/layouts/mailer.html.erb]: Same for img src value in quote here

File: app/views/layouts/mailer.html.erb (hunk 1 above)
Comment (sonalkr132) [reply app/views/layouts/mailer.html.erb]: Thanks :+1: Updated to used double quotes in all src values.

**Pull request merged**